batch/batch/worker/worker.py

```diff
@@ -806,12 +837,13 @@ async def delete_container(self):
         if self.container_is_running():
             try:
                 log.info(f'{self} container is still running, killing crun process')
-                self.process.kill()
+                self.process.terminate()
```

Comment:
huh, I missed the signal. I don't quite know the best way about this, I was finding today that `terminate` wasn't very effective at actually shutting the `crun` process down, and `kill` could stop the `crun` process but not the child processes in the container. `crun kill` was the only option I tried that actually cleared all processes out of the container. Looks like `crun` [defaults to SIGTERM](https://github.com/containers/crun/blob/0b0b2887403de620c501a70db853ba7ebbdede3a/src/kill.c#L103), but I can do so explicitly since it is not documented.